CHANGELOG.md

```diff
@@ -2,9 +2,39 @@
 All notable changes to this project will be documented in this file.
 This project aims to adhere to [Semantic Versioning](http://semver.org/).
 
-## [3.3.2] -
+## [3.4.0] -
+### Added
+ - [Updated Prune-Depth Documentation. Add Unit Tests for more limit values](https://github.com/joyent/java-manta/pull/484)
 ### Fixed
- - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486) 
+ - [Content-Type is not set to application/octet-stream when doing CSE](https://github.com/joyent/java-manta/issues/486)
+ - [Test Groups that include Heavyweight Tests that are not run by default](https://github.com/joyent/java-manta/issues/483) 
+ - [Invalid MantaUser results in 500 Internal Server Error](https://github.com/joyent/java-manta/issues/123)
+ - [Change all hard coded "/" values to MantaClient.SEPARATOR ](https://github.com/joyent/java-manta/issues/363)  
+### Changed
+ - [Upgraded upstream dependencies to latest stable versions:](https://github.com/joyent/java-manta/pull/485)
+     - HTTP Signatures dependency: 4.0.9 → 4.0.10
```

Comment:
Please link to the commit and not dump all of the versions.

Reply:
Done.